### PR TITLE
Unify Twig & Live components

### PR DESCRIPTION
**Pull request by jakubtobiasz**
- `:` convention applied, so every component is named like `sylius_admin:some:component`, so it can be used like `<twig:sylius_admin:some:component />` in Twig templates
- `sylius.live_component` shortcut tag has been added
- all attribute-based autoconfiguration for LiveComponents has been removed

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/config/services/twig/component.xml [hunk 1]
```diff
@@ -11,112 +11,265 @@
...
+                name="sylius.live_component"
+                key="sylius_admin:shipment:ship"
+                template="@SyliusAdmin/shipment/component/ship.html.twig"
+            />
         </service>
 
-        <service id="Sylius\Bundle\AdminBundle\Twig\Component\Order\AddressHistoryComponent">
-            <argument type="service" id="sylius.repository.address_log_entry" />
+        <service
+            id="sylius_admin.twig.component.taxon.tree_component"
```
Comment (Wojdylak) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/twig/component.xml]:
```suggestion
            id="sylius_admin.twig.component.taxon.tree"
```

File: src/Sylius/Bundle/AdminBundle/Resources/config/services/twig/component.xml [hunk 2]
```diff
@@ -11,112 +11,265 @@
...
+
+            <tag
+                name="twig.component"
+                key="sylius_admin:payment_method:form"
+                template="@SyliusAdmin/payment_method/form.html.twig"
+            />
         </service>
 
-        <service id="Sylius\Bundle\AdminBundle\TwigComponent\Product\FormComponent">
+        <service id="Sylius\Bundle\AdminBundle\Twig\Component\Product\FormComponent">
```
Comment (Wojdylak) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/twig/component.xml]:
```suggestion
        <service 
            id="sylius_admin.twig.component.product.form
            class="Sylius\Bundle\AdminBundle\Twig\Component\Product\FormComponent"
       >
```

**Review by Wojdylak — APPROVED** [review]

File: src/Sylius/Bundle/AdminBundle/Twig/Component/Shipment/ShipComponent.php [hunk 3]
```diff
@@ -22,8 +22,8 @@
 use Symfony\UX\LiveComponent\ComponentWithFormTrait;
 use Symfony\UX\LiveComponent\DefaultActionTrait;
 
-#[AsLiveComponent(name: 'SyliusAdmin.Shipment.ShipType', template: '@SyliusAdmin/shipment/component/ship.html.twig')]
-final class ShipType
+#[AsLiveComponent]
+final class ShipComponent
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/AdminBundle/Twig/Component/Shipment/ShipComponent.php]:
```suggestion
final class ShipFormComponent
```
Just to emphasize that it is a form.

**Review by NoResponseMate — APPROVED** [review]

**Review by GSadee — APPROVED** [review]
But the rebase is needed 🖖🏻

**Comment by jakubtobiasz** [comment]
I've rebased the PR, and applied all suggestions c:.

File: src/Sylius/Bundle/AdminBundle/Resources/config/services/twig/component.xml [hunk 4]
```diff
@@ -11,117 +11,282 @@
...
-            <argument type="service" id="sylius.manager.taxon" />
+        <service
+            id="sylius_admin.twig.component.order.address_history"
+            class="Sylius\Bundle\AdminBundle\Twig\Component\Order\AddressHistoryComponent"
+        >
+            <argument type="service" id="sylius.repository.address_log_entry" />
+
+            <tag
+                name="twig.component"
+                key="sylius_admin:order:history:address"
```
Comment (Wojdylak) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/twig/component.xml]:
```suggestion
                key="sylius_admin:order:address_history"
```

**Review by GSadee — APPROVED** [review]

**Pull request merged**